src/components/project/DetailButtonChild/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
There's already `DetailButton` that looks exactly the same. You could have used the same component but with a different style composition. Think variations.